source/frontend/src/features/mapSideBar/layout/MapSideBarLayout.tsx

```diff
@@ -27,101 +26,88 @@ const MapSideBarLayout: React.FunctionComponent<
 > = ({ title, header, icon, showCloseButton, ...props }) => {
   const { mapSideBarViewState, toggleSidebarDisplay } = useMapStateMachine();
   return (
-    <VisibilitySensor partialVisibility={true}>
-      {({ isVisible }: any) => {
-        if (mapSideBarViewState.isCollapsed) {
-          return (
-            <StyledCollapsedSidebarWrapper>
-              <Row>
-                <Col xs={12} className="justify-content-center d-flex">
-                  {icon}
-                </Col>
-              </Row>
-              <Underline className="mb-4" />
-              <Row>
-                <StyledButtonBar xs={12} className="align-items-center d-flex flex-column-reverse">
-                  <TooltipWrapper
-                    tooltipId="expand-sidebar-tooltip-expand"
-                    tooltip={'Expand Screen'}
-                  >
-                    <span>
-                      <ExpandCollapseButton
-                        expanded={!mapSideBarViewState.isCollapsed}
-                        toggleExpanded={toggleSidebarDisplay}
-                      />
-                    </span>
-                  </TooltipWrapper>
-                  {showCloseButton && (
-                    <TooltipWrapper tooltipId="close-sidebar-tooltip" tooltip="Close Form">
-                      <CloseIcon title="close" onClick={props.onClose} />
-                    </TooltipWrapper>
-                  )}
-                </StyledButtonBar>
-              </Row>
-            </StyledCollapsedSidebarWrapper>
-          );
-        }
-        return (
-          <StyledSidebarWrapper>
-            <Row>
-              <Col>
-                <Styled.H1 className="mr-auto">
-                  <>
-                    <span className="mr-2">{icon}</span>
-                    {title}
-                  </>
-                </Styled.H1>
-              </Col>
-
-              <StyledButtonBar xs="auto" className="d-flex">
-                <TooltipWrapper
-                  tooltipId="expand-sidebar-tooltip-collapse"
-                  tooltip={'Collapse Screen'}
-                >
-                  <span>
-                    <ExpandCollapseButton
-                      expanded={!mapSideBarViewState.isCollapsed}
-                      toggleExpanded={toggleSidebarDisplay}
-                    />
-                  </span>
+    <StyledSidebarWrapper className={mapSideBarViewState.isCollapsed ? '' : 'expanded'}>
```

Comment:
nit: this could also be passed as a parameter to the styled component

Reply:
The other subtle change here is that we removed the **VisibilitySensor** (which we mock in a lot of unit tests). Given that we don't use it anymore we should update the tests to remove the unnecessary mocking.
Not required for this PR - should be a tech-debt task. Just calling out the change